modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/list/List.tsx

```diff
@@ -3,12 +3,148 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import ClayAlert from '@clayui/alert';
+import {ClayButtonWithIcon} from '@clayui/button';
+import {Body, Cell, Head, Row, Table} from '@clayui/core';
+import {ClayInput} from '@clayui/form';
+import ClayLayout from '@clayui/layout';
+import classNames from 'classnames';
+import {openModal} from 'frontend-js-web';
 import React from 'react';
 
+import {IFDSViewSectionProps} from '../../../FDSView';
 import {IBaseVisualizationMode} from '../VisualizationModes';
+import {IFDSField} from '../table/Table';
+import AddFieldsModalContent from '../table/modal_content/AddFieldsModalContent';
 
 export interface IList extends IBaseVisualizationMode<'list'> {}
 
-export default function List() {
-	return <>List</>;
+interface IListField {
+	fieldId: string;
+	label: string;
+}
+
+const LIST_VISUALIZATION_MODE_FIELDS: IListField[] = [
+	{fieldId: 'title', label: Liferay.Language.get('title')},
+	{fieldId: 'description', label: Liferay.Language.get('description')},
+	{fieldId: 'symbol', label: Liferay.Language.get('symbol')},
+	{fieldId: 'link', label: Liferay.Language.get('link')},
+	{fieldId: 'label', label: Liferay.Language.get('label')},
+];
+
+const LABEL_CELL_WIDTH = '30%';
```

Comment:
This should probably be in a CSS file, so it's more convenient to override for customers. Also, I am not a fan of percentage widths, they are inflexible to content and sometimes needlessly break rows. Would something like `min-width: 30%` work?

Reply:
I agree, this should be CSS but... I tried and Clay expects this to be specified as a prop (maybe because of being a table cell, but still a bit weird).

If I try to set this with a CSS selector, Clay overrides it's value by updating width as a style attribute:

![image](https://github.com/liferay-frontend/liferay-portal/assets/800645/2d517172-45a9-4d5e-85ab-da10db4da9ac)

So even if I use min-width instead of width, all columns are set to auto:

![image](https://github.com/liferay-frontend/liferay-portal/assets/800645/aaf588c3-9564-4db9-87be-5d2d16014d7a)

In the design the left column is much smaller, and I tried to do this more dynamic relying on CSS grid, but using a table layout and then changing the styles to behave like a grid felt wrong...

Anyway that's all the process I followed :sweat_smile:. I'd be happy to look for another solution if you have any idea.

Reply:
I didn't notice it before, but you are using 
```js
import {Body, Cell, Head, Row, Table} from '@clayui/core';
```
I don't see import like this anywhere else in DXP. Maybe these components are not meant to me used directly, resulting in quirky width API.

In many places we import 
```js
import ClayTable from '@clayui/table';
```
that exposes `<ClayTable.Cell>` etc.

Reply:
Oh, I didn't know that. Let me update this and see if it works better...

Reply:
Nah, it works the same, probably those are just aliases. Looking for other ways to solve it I show that the Clay docs use the @clay/core imports (I think I copy/pasted from there, but I don't remember):

https://clayui.com/docs/components/table.html

![image](https://github.com/liferay-frontend/liferay-portal/assets/800645/81d03969-698d-41b8-8daf-193be74f42e7)

Reply:
> probably those are just aliases

Looks line no, they are completely separated implementations. [ClayTable](https://github.com/liferay/clay/blob/master/packages/clay-table/src/index.tsx) vs [core Table](https://github.com/liferay/clay/blob/master/packages/clay-core/src/table/Table.tsx). I have no idea why both of these exist.

We should definitely use `ClayTable`, as that is consistent with DXP. Unfortunately, that doesn't solve the width problem. In the end, if you cannot cook up a more elegant solution, we can live with percentage width. It is not a blocker.